packages/edit-site/src/components/layout/index.js

```diff
@@ -153,13 +153,13 @@ export default function Layout( { route } ) {
 										} }
 										className="edit-site-layout__sidebar"
 									>
-										<SiteHub
-											ref={ toggleRef }
-											isTransparent={
-												isResizableFrameOversized
-											}
-										/>
 										<SidebarContent routeKey={ routeKey }>
+											<SiteHub
```

Comment:
Unfortunately it looks like moving the `SiteHub` to be a child of `SidebarContent` means that the side bar content's focusing behaviour now matches against the buttons within the site hub, which causes one of the e2e tests to fail (`'Can use keyboard to navigate the site editor').

From testing manually it seems that what's happening is that if you navigate to the Pages button from the root and hit Enter, when you're taken to the pages screen, the W icon is focused instead of the back button. I.e. here's where focus now lands:

<img width="385" alt="image" src="https://github.com/WordPress/gutenberg/assets/14988353/ed58b8ac-e760-4978-a375-82e124562fc1">

Here's the function that handles the focusing: https://github.com/WordPress/gutenberg/blob/4866c3b3d8d331904fe5b4bc45737077d9322c92/packages/edit-site/src/components/sidebar/index.js#L22

I'm wondering if we need to either have a way for that focusing to skip the site hub and just look at the sidebar area, or if we need a way to give the site hub access to the navigation context without the other wrapping logic?

Reply:
I've pushed a commit in https://github.com/WordPress/gutenberg/pull/63060/commits/4401f6037f19563d60fe50d3b1e21add43a106e3 that hopefully addresses the focus issue 🤞. It's a little hacky but let's see if it gets the test passing.